lib/src/create_report.dart

```diff
@@ -845,6 +845,10 @@ Future<ReportSection> _multiPlatform(String packageDir, Pubspec pubspec) async {
         'runtime:web': 'js',
       };
 
+      // We don't want the native-aut runtime to be explained here.
+      explanations.removeWhere(
+          (explanation) => explanation.tag == 'runtime:native-aot');
```

Comment:
nit: would be nice to reference this via a const value.

Reply:
Done